lua/textcase/plugin/plugin.lua

```diff
@@ -274,7 +274,11 @@ function M.visual(case_method)
 
   if M.state.telescope_previous_visual_region ~= nil then
     utils.set_visual_region(M.state.telescope_previous_visual_region)
-    vim.api.nvim_feedkeys("gvg@", "i", false)
+    if M.state.telescope_previous_visual_region.mode == constants.visual_mode.BLOCK then
+      vim.api.nvim_feedkeys("g@`[", "i", false)
```

Comment:
I know this notation is used in several plugins, but it is not readable. 
`g@` executes what is in `vim.o.operatorfunc`
I barely remember the diff between ``` `[ ``` , ``` `> ``` and ``` `< ```, from the docs:

`''`    Jump back to the last line where the cursor was in the current buffer before jumping
``` `` ```    Jump back to the last position in the current buffer before jumping
``` `[ ```    Jump to the beginning of the previously changed/pasted text
``` `] ```   Jump to the end of the previously changed/pasted text
``` `< ```   Jump to the beginning of the last visual selection
``` `> ```    Jump to the end of the last visual selection
``` `0 ```   Jump back to the last edited file when exiting Vim

Reply:
Mhh, from the docs it seems the wrong thing to do ...

Reply:
Nvim quirks. It wasn't working anyway. 

Tested the branch manually on my local and it is working